kasa/tests/fakeprotocol_smart.py

```diff
@@ -223,6 +236,9 @@ def _send_request(self, request_dict: dict):
             return retval
         elif method == "set_qs_info":
             return {"error_code": 0}
+        elif method == "set_dynamic_light_effect_rule_enable":
+            self._set_dynamic_light_effect_rule_enable(info, params)
```

Comment:
Maybe just use `self._set_effect` to keep it simple? We don't need to follow the conventions used by the internal api here, so `set_effect` would be clearer, IMO.

Reply:
Done, went for `_set_light_effect`